sdk/documentintelligence/ai-document-intelligence-rest/README.md

```diff
@@ -21,6 +21,7 @@ This table shows the relationship between SDK versions and supported API version
 
 | SDK version  | Supported API version of service |
 | ------------ | -------------------------------- |
+| 1.0.0        | 2024-11-30                       |
 | 1.0.0-beta.3 | 2024-07-31-preview               |
 | 1.0.0-beta.2 | 2024-02-29-preview               |
 | 1.0.0-beta.1 | 2023-10-31-preview               |
```

Comment:
These should be removed, right?
```suggestion
```

Reply:
The other languages haven't removed either, and they could be a quick reference for customers too.
I'd keep them.

Reply:
Preview endpoints are typically deprecated and become offline much sooner than GA ones. When this happens, it would be awkward for our users who download this stable version to still see entries about the gone preview endpoints in the README file.

Reply:
ok, I'll remove